packages/suite-desktop/src-electron/modules/custom-protocols.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * Support custom protocols (for example: `bitcoin:`)
+ */
+import { app } from 'electron';
+import { isValidProtocol } from '@lib/protocol';
+
+const init = ({ mainWindow }: Dependencies) => {
+    const { logger } = global;
+
+    if (!process.env.PROTOCOLS) {
+        logger.warn('custom-protocols', 'No protocols defined');
+        return;
+    }
+
+    // For some reason, PROTOCOLS is an array of strings but is considered as a string by TS...
```

Comment:
it is because `process.env` returns an object with all env variables user has in the system. afaik the value is always string (as bash/shells doesn't really have any other data type).  Now I am wondering what will happen if user has already set PROTOCOLS env var, does custom `define` in https://github.com/trezor/trezor-suite/pull/2879/files#diff-de765f1259dbaf301a222fb9435bbedd6cb928f8dd82c56cf27a9fa6dbf12bc2R32 take precedence? Most likely yes, but if not this could crash the app 😁

Reply:
esbuild transforms them at build time, no risk of collision for these :slightly_smiling_face: 
![Screenshot from 2021-02-01 13-46-22](https://user-images.githubusercontent.com/30026749/106460677-fba81980-6493-11eb-8710-38deea6bdd73.png)

Reply:
I know, I am just explaining why TS expects a string, not an array :) 
Now it hit me, maybe more proper way to do this would be to define new global var `PROTOCOLS` instead of polluting process.env? It doesn't have anything to do with env vars anyway. I assume that would require some additional global type to let TS know about the variable, but at least they would be correct.

Reply:
Sorry I accidentally skipped to the last part of your comment :sweat_smile: Yeah that makes total sense. I'll update that.